src/styles/colors.scss

```diff
@@ -0,0 +1,7 @@
+$color-white: #ffffff;
+$color-light-grey: #555555;
+$color-medium-grey: #424242;
+$color-dark-grey: #232323;
+$color-red: #f65261;
+$color-black: rgb(0, 0, 0);
```

Comment:
There's no point in having variable names match the color name - imagine the simple scenario where the customer asks to change the "main" color of the application from blue to red. Instead of changing just one value in the variable, you'd need to update the variable name as well its all references.

For now, it can stay - although in the future I'd strongly discourage this and use other conventions like

```scss
$primary-branding-color: blue;
$secondary-branding-color: lightblue;
$main-text-color: black;
/* etc */
```

Reply:
Good point. I updated the names of variables.